code/modules/admin/verbs/randomverbs.dm

```diff
@@ -979,6 +979,9 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(!check_rights(R_ADMIN))
 		return
 
+	var/mob/M = tgui_input_list(src.mob, "Which player would you like to cryo?", "Pick to cryo", living_mob_list)
+
+
```

Comment:
tiny not-very-important thing: no need for two empty lines here aside from that this is fine to merge

Reply:
What's with your aversion against empty lines?

Reply:
mostly that they look kinda messy if inserted unnecessarily